auditbeat/docs/setting-up-running.asciidoc

```diff
@@ -4,7 +4,7 @@
 // that is unique to each beat.
 /////
 
-[[seting-up-and-running]]
```

Comment:
Seems like we used this link knowwhere and is the reason this worked.

Reply:
Those things appear to be https://www.elastic.co/guide/en/beats/filebeat/current/seting-up-and-running.html#seting-up-and-running (and similar?)

Reply:
@dedemorton Will this change break things?

Reply:
@ruflin There's a small change that it could break the doc build (if there's another book that links to this topic), but it would be easy to fix. 

The biggest impact is that the ID controls the html page file name, which means that I would need to get redirects set up for any branches that have this change.

I think it looks bad to have a typo in the URL, though, so I would go ahead with this change.